unity/Assets/StreamingAssets/content/MoM/ck/mom1e-tiles/tiles.ini

```diff
@@ -78,6 +78,7 @@ reverse=TileSideLaboratory
 [TileSideFrontYard]
 name={ffg:TILE_FRONT_YARD_MAD01}
 image="img/Tile_Front_Yard_MAD01"
+aspect=1.5
```

Comment:
This still isn't needed, while it probably won't do any hard I'd rather not have it.  Otherwise OK to merge.